examples/kind-multinode-kcp/egress_selector_configuration.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiserver.k8s.io/v1beta1
+kind: EgressSelectorConfiguration
+egressSelections:
+- name: cluster
+  connection:
+    proxyProtocol: GRPC
+    transport:
+      uds:
+        udsName: /etc/kubernetes/konnectivity-server/konnectivity-server.socket
+- name: master
```

Comment:
this is deprecated in favor of 'controlplane'

Reply:
+1